src/force/CabanaPD_ForceModels_PMB.hpp

```diff
@@ -48,7 +49,7 @@ struct ForceModel<PMB, Elastic, TemperatureIndependent> : public BaseForceModel
     {
         delta = _delta;
         K = _K;
-        c = 18.0 * K / ( 3.141592653589793 * delta * delta * delta * delta );
+        c = 18.0 * K / ( pi * delta * delta * delta * delta );
```

Comment:
Why "c" is defined both here and in CabanaPD_Inputs.hpp (Line 129)?

Reply:
It's definitely needed for the forces here.. and in the inputs it's for the timestep determination. We could define a free function and use it for both